style/ruby/README.md

```diff
@@ -3,47 +3,51 @@ Ruby
 
 [Sample](sample.rb)
 
-* Avoid conditional modifiers (lines that end with conditionals).
-* Avoid multiple assignments per line (`one, two = 1, 2`).
-* Avoid organizational comments (`# Validations`).
+* Avoid conditional modifiers (lines that end with conditionals). 36491dbb9
```

Comment:
Does the short SHA auto-link? Docs don't list it but would be nice if so.

https://help.github.com/articles/autolinked-references-and-urls/

Doesn't look like short SHA or pound-number working in "View" on this PR but pound-number listed in docs as autolinking.